src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -65,35 +65,29 @@
 
   "minecolonies.config.initialcitizenamount": "Initital Citizen Count",
   "minecolonies.config.initialcitizenamount.comment": "Amount of initial citizens.",
-  "minecolonies.config.builderplaceconstructiontape": "Contruction-Tape Placement",
-  "minecolonies.config.builderplaceconstructiontape.comment": "Should the Builder place construction tape?",
-  "minecolonies.config.playergetsguidebookonfirstjoin": "Guidebook",
-  "minecolonies.config.playergetsguidebookonfirstjoin.comment": "Should a player get one guidebook on their first join to a new world?",
+  "minecolonies.config.builderplaceconstructiontape": "Contruction Tape Placement",
+  "minecolonies.config.builderplaceconstructiontape.comment": "Should construction tape be placed around in-construction builds?",
   "minecolonies.config.allowinfinitesupplychests": "Infinite Supply Camps/Ships",
   "minecolonies.config.allowinfinitesupplychests.comment": "Should players be able to place an infinite amount of supply camps/ships?",
   "minecolonies.config.allowinfinitecolonies": "Infinite Colonies",
   "minecolonies.config.allowinfinitecolonies.comment": "Should players be allowed to abandon their colony to create a new one easily? Note: This is buggy!",
   "minecolonies.config.allowotherdimcolonies": "Other Dim Colonies",
-  "minecolonies.config.allowotherdimcolonies.comment": "Should colonies in other dimensions be allowed? Note: This is buggy if the dimension doesn't have day/night!",
-  "minecolonies.config.citizenrespawninterval": "Citizen Respawn Interval",
-  "minecolonies.config.citizenrespawninterval.comment": "Average citizen respawn interval (in seconds).",
+  "minecolonies.config.allowotherdimcolonies.comment": "Should colonies in other dimensions be allowed?",
+  "minecolonies.config.citizenrespawninterval": "Inital Citizen Respawn Interval",
+  "minecolonies.config.citizenrespawninterval.comment": "If a citizen dies and there are less than four citizens left, this changes the amount of time before another citizen spawns.",
```

Comment:
Not sure if it's worth spelling out, but citizensrespawninterval applies for all (sane) settings of initialcitizenamount, not just the default four.

Reply:
`  "minecolonies.config.citizenrespawninterval.comment": "If a citizen dies and there are less than the initial number of citizens left, this changes the amount of time before another citizen spawns.",` maybe? Bit wordy, hmm.

Reply:
Wordy, but probably clearer than referencing another variable directly.  And it does end up spaced apart by the configbuilder.  If you want to snip it down, this does effect the first citizens spawn rate as well, so : 

`"minecolonies.config.citizenrespawninterval.comment": "If a colony is under the initial citizen count, this changes the amount of time before a new citizen spawns.", `

Reply:
Oh first citizen spawn rate too? I'll go with that then